ldb/export.py

```diff
@@ -158,7 +158,7 @@ def set_fields(self, query):
     allowed_querysets = {
         'organizations': Q(~Q(organization__isnull=True)),
         'members': Q(living_person, Q(person__member__date_from__isnull=False),
-                     Q(person__member__date_to__isnull=True)),
+                     Q(person__member__date_to__gt=date.today())),
```

Comment:
Geen zin om te testen :angel:, dit werkt ook gewoon als de `date_to` wel null is?